auditlog_security/views/auditlog_view.xml

```diff
@@ -73,24 +107,11 @@
                                     <field name="auditlog_rule_id" readonly="1" invisible="1"/>
                                     <field name="group_ids"
                                         attrs="{'readonly': [('state', '=', 'subscribed')]}"
-                                        >
-                                    <tree editable="bottom">
```

Comment:
@thomaspaulb also this was responsible for the pop up _onchange_ errors Juan found.So It is just using the default view with no issues.